orchestration/roles/app/tasks/setup_django_app.yml

```diff
@@ -1,7 +1,10 @@
 ---
 
 - name: Install packages required by the Django app inside virtualenv
-  pip: virtualenv={{ virtualenv_path }} requirements={{ requirements_file }}
+  pip: virtualenv={{ virtualenv_path }} requirements={{ requirements_file }} executable="{{ virtualenv_path }}/bin/pip3"
+
+- name: Make `manage.py` executable
+  file: path="{{project_path}}/manage.py" mode="u+x,g+x"
```

Comment:
should we keep consistency with spaces between the curly braces and the variable here and elsewhere you added them?